modules/react/icon/lib/AppletIcon.tsx

```diff
@@ -67,23 +79,56 @@ export interface AppletIconProps extends AppletIconStyles, Pick<IconProps, 'shou
   size?: number;
 }
 
+export const appletIconStencil = createStencil({
+  extends: svgStencil,
+  vars: {
+    color200: '',
+    color300: '',
+    color400: '',
+    color500: '',
+  },
+  base: ({color200, color300, color400, color500, size}) => ({
+    [size]: px2rem(92),
+    '& .color-100': {
+      fill: base.frenchVanilla100,
+    },
+    '& .color-200': {
+      fill: cssVar(color200, base.blueberry200),
+    },
+    '& .color-300': {
+      fill: cssVar(color300, base.blueberry300),
+    },
+    '& .color-400': {
+      fill: cssVar(color400, base.blueberry400),
+    },
+    '& .color-400-alpha-20': {
+      fill: cssVar(color400, base.blueberry400),
+    },
+    '& .color-500': {
+      fill: cssVar(color500, base.blueberry400),
+    },
+  }),
+});
+
 export const AppletIcon = createComponent('span')({
   displayName: 'AppletIcon',
-  Component: (
-    {size = 92, icon, color, shouldMirror, ...elemProps}: AppletIconProps,
-    ref,
-    Element
-  ) => {
+  Component: ({size, icon, color, ...elemProps}: AppletIconProps, ref, Element) => {
+    const colors = color && {
+      color200: base[`${color}200` as keyof typeof base],
+      color300: base[`${color}300` as keyof typeof base],
+      color400: base[`${color}400` as keyof typeof base],
+      color500: base[`${color}500` as keyof typeof base],
```

Comment:
I think it is better to use `as const` if you want to narrow the type:

```suggestion
      color200: base[`${color}200` as const],
      color300: base[`${color}300` as const],
      color400: base[`${color}400` as const],
      color500: base[`${color}500` as const],
```